roles/openshift_deploy_registry/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+# defaults file for openshift_deploy_registry
```

Comment:
I'd personally like to see us not commit empty files in roles, but I know @twiest disagrees with me on that point.